Src/Plugins/Security/SEC101_038.PostgreSqlConnectionStringValidator.cs

```diff
@@ -61,21 +59,15 @@ protected override IEnumerable<ValidationResult> IsValidStaticHelper(Dictionary<
                 return ValidationResult.CreateNoMatch();
             }
 
-            if (host.Value == "tcp")
-            {
-                return ValidationResult.CreateNoMatch();
-            }
-
             FlexMatch unused = null;
             FlexMatch matchedPattern = groups["0"];
             string port = ParseExpression(RegexEngine, matchedPattern, PortRegex, ref unused);
             string database = ParseExpression(RegexEngine, matchedPattern, DatabaseRegex, ref unused);
 
             string hostValue = FilteringHelpers.StandardizeLocalhostName(host.Value);
 
-            ValidationState exclusionResult = FilteringHelpers.HostExclusion(hostValue, HostsToExclude);
-
-            if (exclusionResult == ValidationState.NoMatch)
+            if (HostsToExclude.Contains(hostValue) ||
+                hostValue.IndexOf("mysql", StringComparison.OrdinalIgnoreCase) != -1)
```

Comment:
>hostValue.IndexOf("mysql", StringComparison.OrdinalIgnoreCase) [](http://example.com/codeflow?start=16&length=62)

I think we should not do this.
If we find a mysql IP address or something that isn't azure related, we would drop.